map.json

```diff
@@ -0,0 +1,117 @@
+[
+    {
+        "version": 2,
+        "path": "nvram3.bin",
+        "entries": [
+            {
+                "name": "_version",
+                "offset": 53,
+                "size": 1
+            },
+            {
+                "name": "save3_bool",
+                "offset": 54,
+                "size": 1,
+                "bit_offset": 0,
+                "is_bool": true
+            },
+            {
+                "name": "save3_bool2",
+                "offset": 54,
+                "size": 1,
+                "bit_offset": 1,
+                "is_bool": true
+            },
+            {
+                "name": "save3_byte",
+                "offset": 55,
+                "size": 1
+            },
+            {
+                "name": "save3_int",
+                "offset": 56,
+                "size": 4
+            },
+            {
+                "name": "save3_double",
+                "offset": 60,
+                "size": 8
+            },
+            {
+                "name": "save3_string",
+                "offset": 68,
+                "size": 2
+            },
+            {
+                "name": "save3_irange",
+                "offset": 70,
+                "size": 16
+            },
+            {
+                "name": "save3_drange",
+                "offset": 86,
+                "size": 32
```

Comment:
It's just a lexical scanner, so, yes, the order that things will be processed are the order that they're specified in the JSON file. Since every field is an object inside an array, the order shouldn't be changed by other tools (beautifiers, etc) as well.